manifests/components/kibana.jsonnet

```diff
@@ -67,7 +67,7 @@ local strip_trailing_slash(s) = (
               env_+: {
                 KIBANA_ELASTICSEARCH_URL: $.es.svc.host,
 
-                local route = $.ingress.spec.rules[1].http.paths[0],
+                local route = $.ingress.spec.rules[0].http.paths[0],
```

Comment:
Isn't there a way to make this more robust than using an index? This is not immediately obvious to the casual reader. So at least it deserves a nice comment.

Reply:
replaced with something more explicit